package.json

```diff
@@ -53,7 +53,7 @@
     "yargs": "^13.2.2"
   },
   "devDependencies": {
-    "@babel/core": "^7.0.0-beta.41",
+    "@babel/core": "^7.5.5",
```

Comment:
Maybe we should also update `preset-env` and `preset-typescript` to latest version?